contracts/vaults/Deposit.sol

```diff
@@ -58,19 +58,17 @@ contract Deposit is Base {
     }
 
     /**
-     * @notice rewardDepositors - add pool tokens to reward depositors in the pool's native token0
+     * @notice rewardDepositors - add pool tokens to reward depositors in the pool's native token
      * The funds will be given to depositors pro rata upon withdraw
      * The sender of the transaction must have approved the spend before calling this function
      * @param _pid pool id
      * @param _amount amount of pool's native token to add
     */
-    function rewardDepositors(uint256 _pid, uint256 _amount) external nonReentrant {
-        if ((poolInfos[_pid].balance + _amount) / MINIMUM_DEPOSIT >=
```

Comment:
Ok, so this was not good after all, see the my comment here: https://github.com/hats-finance/hats-contracts/issues/226#issuecomment-1161530748